### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yaomtc**
Added clarification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Download:
 
 ```bash
 git clone https://github.com/DavidRisch/steamvr_utils.git
+cd ./steamvr-utils
```
Comment (DavidRisch) [inline README.md]: The dash should be an underscore.

**Review by DavidRisch — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (yaomtc) [reply README.md]: Oops, thanks!

**Review by yaomtc — COMMENTED** [review]
EDIT: disregard

**Review by DavidRisch — APPROVED** [review]

**Comment by DavidRisch** [comment]
Thanks for your contribution @yaomtc!

**Pull request merged**